src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -1562,5 +1564,37 @@ public Location get(LootGenerateEvent event) {
 				}
 			}, EventValues.TIME_NOW);
 		}
+		// EntityResurrectEvent
+		EventValues.registerEventValue(EntityResurrectEvent.class, ItemStack.class, new Getter<ItemStack, EntityResurrectEvent>() {
+			@Override
+			@Nullable
+			public ItemStack get(EntityResurrectEvent event) {
+				EquipmentSlot hand = event.getHand();
+				EntityEquipment equipment = event.getEntity().getEquipment();
+				if (equipment == null)
+					return null;
+
+				if (hand == EquipmentSlot.HAND) {
+					return equipment.getItemInMainHand();
+				} else if (hand == EquipmentSlot.OFF_HAND) {
+					return equipment.getItemInOffHand();
+				}
+				return null;
+			}
+		}, EventValues.TIME_NOW);
```

Comment:
You shouldn't need the itemtype when slot is present. The converters should do `event-item`

Reply:
oh cool i didn't realize it works like that!